Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -1277,10 +1277,11 @@ PhysicalParticleContainer::AddPlasma (PlasmaInjector const& plasma_injector, int
                 // With only 1 mode, the angle doesn't matter so
                 // choose it randomly.
                 const Real theta = (nmodes == 1 && rz_random_theta)?
-                    (2._rt*MathConst::pi*amrex::Random(engine)):
-                    (2._rt*MathConst::pi*r.y + theta_offset);
-                pos.x = xb*std::cos(theta);
-                pos.y = xb*std::sin(theta);
+                    (2._rt*amrex::Random(engine)):
+                    (2._rt*r.y + tt_offset);
```

Comment:
Where did the pi go?

Reply:
I am using `amrex::Math::sincospi` instead of `amrex::Math::sincos`, so the `pi` is already taken into account in the function. That's why I wanted to use `tt` instead of `theta` as the name of the variable, in order to avoid confusion. However, I can add back the `pi` and use `amrex::Math::sincos` (it may be more readable).